libnl-route-3.sym

```diff
@@ -1292,6 +1292,7 @@ global:
 	rtnl_link_bridge_set_vlan_stats_enabled;
 	rtnl_link_inet6_get_conf;
 	rtnl_link_info_ops_get;
+	rtnl_link_get_perm_addr;
```

Comment:
3.8 is already released. That means the section "libnl_3_8" must not be changed anymore.

Add a new `libnl_3_9` section at the bottom of the file.